money-api/src/test/scala/com/comcast/money/api/DefaultSpan.scala

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2012-2015 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.comcast.money.api
+
+import java.lang
+import java.lang.{ Double, Long }
+
+import scala.collection._
+import scala.collection.JavaConversions._
+
+// obviously not threadsafe and very mutable, basing this kinda on what I did for the java-only core
+class DefaultSpan(
+    val id: SpanId,
+    val spanName: String,
+    spanMonitor: SpanMonitor,
+    notes: mutable.Map[String, Note[_]] = new mutable.HashMap[String, Note[_]],
+    propagate: Boolean = false
+) extends Span {
+
+  // TODO: yea, hacky, just prototyping though
+  private var stopTime: Long = _
+  private var stopInstant: Long = _
+  private var startedTime: Long = _
+  private var startInstant: Long = _
+  private var success: Boolean = true
+
+  override def startTime(): Long = startedTime
+
+  override def start(): Unit = {
+    startedTime = System.currentTimeMillis
+    startInstant = System.nanoTime
+
+    // Start watching me
+    spanMonitor.watch(this)
```

Comment:
What do you think of keeping the Span management outside of the Span implementation.

The default implementation of Span management should continue to be the akka implementation we have in master, providing us facilities to handle things like backoff and graceful failure.  I see this implementation being best suited for the vast majority of our potential users who will most likely be using Money for instrumentation of Services running in an environment where library footprint is not a concern.

We can bundle a separate implementation in a separate sub-module or project for the few users who are concerned with library footprint or who might be interested in managing span lifecycle on their own.  They would get the more limited Monitor implementation of Span management giving up some resiliency and fault tolerance for a smaller library footprint.